### PR TITLE
[kube-prometheus-stack] Add evaluationInterval to Thanos Ruler

### PR DESCRIPTION
**Pull request by heliapb**
#### What this PR does / why we need it
Add `.thanosRuler.thanosRulerSpec.evaluationInterval ` to thanos ruler templates.
#### Which issue this PR fixes
- fixes #

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
Please raise chart minor version.

**Comment by heliapb** [comment]
@monotek done, could you please review it again?

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.7.0
+version: 39.7.1
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:25]:
```suggestion
version: 39.8.0
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/templates/thanos-ruler/ruler.yaml [hunk 2]
```diff
@@ -44,6 +44,7 @@ spec:
   logFormat: {{ .Values.thanosRuler.thanosRulerSpec.logFormat | quote  }}
   logLevel:  {{ .Values.thanosRuler.thanosRulerSpec.logLevel | quote  }}
   retention: {{ .Values.thanosRuler.thanosRulerSpec.retention | quote  }}
+  evaluationInterval: {{ .Values.thanosRuler.thanosRulerSpec.evaluationInterval | quote }}
```
Comment (monotek) [inline charts/kube-prometheus-stack/templates/thanos-ruler/ruler.yaml:47]: If the default is not set this should not be applied at all.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 1 above)
Comment (heliapb) [reply charts/kube-prometheus-stack/Chart.yaml]: Done, could you please check again, thanks

File: charts/kube-prometheus-stack/templates/thanos-ruler/ruler.yaml (hunk 2 above)
Comment (heliapb) [reply charts/kube-prometheus-stack/templates/thanos-ruler/ruler.yaml]: Done, could you please check again, thanks

**Review by monotek — APPROVED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
Please fix trailing spaces

**Comment by heliapb** [comment]
Changed, please review. Thanks

**Review by monotek — APPROVED** [review]

**Pull request merged**